### PR TITLE
master versions of add-on manager manifests

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

**Comment by caseydavenport** [comment]
@simonswine FYI 

I've ported the manifests over to the `master/` tree (which is where we cut new releases from) and moved the manifests into a subdirectory of the `k8s-backend` directory.

**Comment by djosborne** [comment]
What's the plan with this PR?

**Comment by caseydavenport** [comment]
@djosborne I need to find a reviewer, and then merge.  Care to have a go?

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 1]
```diff
@@ -32,14 +34,27 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
...
+YAML manifests from the `/etc/kubernetes/addons` directory and ensures they
+exist in the cluster. The following lines need to be run on the master of
+Kubernetes cluster with addon-manager installed:
+
+```
+curl -sL -o /etc/kubernetes/addons/calico-configmap.yaml http://docs.projectcalico.org/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/addon-manager/calico-configmap.yaml
+curl -sL -o /etc/kubernetes/addons/calico-daemonset.yaml http://docs.projectcalico.org/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/addon-manager/calico-daemonset.yaml
+```
+
 #### Example: kubeadm + flannel
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: Should this really be a subsection of addon-manager?

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 2]
```diff
@@ -32,14 +34,27 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
 
 Below are a few examples for how to get started.
 
+### Using Kubernetes' addon-manager
+
+Kubernetes' [addon-manager](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager)
+can be used for deploying Calico alongside with other cluster addons. It takes
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: how bout

The addon-manager monitors `/etc/kubernetes/addons` for YAML manifests and ensures they are applied to the cluster.

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 3]
```diff
@@ -32,14 +34,27 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
...
+YAML manifests from the `/etc/kubernetes/addons` directory and ensures they
+exist in the cluster. The following lines need to be run on the master of
+Kubernetes cluster with addon-manager installed:
+
+```
+curl -sL -o /etc/kubernetes/addons/calico-configmap.yaml http://docs.projectcalico.org/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/addon-manager/calico-configmap.yaml
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: Can't be just one file for both?

Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: The add-on manager requires that separate objects go in separate files.

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md (hunk 1 above)
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: Nope.

**Comment by caseydavenport** [comment]
@djosborne this is ready for another look.

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 4]
```diff
@@ -34,6 +36,21 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
 
 Below are a few examples for how to get started.
 
+### Using Kubernetes' addon-manager
+
+The Kubernetes [addon-manager](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager)
+can be used for deploying Calico alongside with other cluster addons. Monitors
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: - remove 'with'
- add 'It' before 'Monitors'

File: master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md [hunk 5]
```diff
@@ -34,6 +36,21 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
...
+`/etc/kubernetes/addons` for manifests and ensures they
+are applied to the cluster. The following lines need to be run on the master of
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md]: referring to them as lines is a bit weird. Can we do:

"Run the following commands on a kubernetes master which has the addon-manager installed:"

**Comment by djosborne** [comment]
2 more nits I noticed. Then do these need to be put into v2.1 as well? Otherwise, LGTM

**Comment by caseydavenport** [comment]
@djosborne I think this is ready for another look.

**Comment by djosborne** [comment]
Can you update the docs to explain these are k8s 1.5 manifests? And arrange them in a way so that there is room to add 1.6 versions of them?

**Pull request closed without merging**